inc/namespace.php

```diff
@@ -590,12 +590,30 @@ function migrate_wpseo_to_yoast() : void {
 
 			// Handle post meta.
 			$posts_query_args = [
-				'post_type' => 'any',
+				// Get all post types, even those with exclude_from_search (so "any" is not appropriate https://core.trac.wordpress.org/ticket/17592).
+				'post_type' => get_post_types(),
 				'post_status' => 'any',
 				'posts_per_page' => 0,
-				'meta_key' => '_meta_title',
-				'meta_compare' => 'EXISTS',
 				'fields' => 'ids',
+				// Ignore filters that may exclude certain post types from queries.
+				'suppress_filters' => true,
+				// phpcs:disable HM.Performance.SlowMetaQuery.dynamic_query -- check required for all of this meta. Performance here isn't a major issue, as this is only run manually on upgrade.
+				'meta_query' => [
```

Comment:
This also shouldn't technically trigger the linter because `EXISTS` queries don't have the same performance issues as other types of meta query :)

Reply:
HM Linter should be allowing this!